src/PrestaShopBundle/Service/DataProvider/Marketplace/ApiClient.php

```diff
@@ -100,7 +100,7 @@ public function getCategories()
 
         $responseArray = json_decode($response);
 
-        return $responseArray->module;
+        return isset($responseArray->module)?$responseArray->module:array();
```

Comment:
I have a slow connexion, I would prefer to have the smallest files as possible 😆